### DISCUSSION TITLE
Cozinhando com a Sétima Arte

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#435] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/cozinhando-com-a-setima-arte/

**Comment by one-eyed-willy** [comment]
![null](<https://i.imgur.com/PDGhVKG.png>)

**Comment by angellisy** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 20 de junho de 2022](<https://www.vish.blog.br/imagens/cozinhando-com-a-setima-arte/#comment_1160649671>):

> [![null](<https://camo.githubusercontent.com/bd0b0ef07a1e46bab65ed4060147e07a311f4b0415db9a8b8563ab6fc52a8684/68747470733a2f2f692e696d6775722e636f6d2f50444768564b472e706e67>)](<https://camo.githubusercontent.com/bd0b0ef07a1e46bab65ed4060147e07a311f4b0415db9a8b8563ab6fc52a8684/68747470733a2f2f692e696d6775722e636f6d2f50444768564b472e706e67>)

Esse filme é lindo! :smiling\_face\_with\_three\_hearts:

**Comment by maryon-telvanni** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [segunda-feira, 20 de junho de 2022](<https://www.vish.blog.br/imagens/cozinhando-com-a-setima-arte/#comment_1160681301>):

> Esse filme é lindo! 🥰

O som dos pequeninos andando, pegando os objetos... É tudo tão... relaxante
